components/automate-ui/src/app/services/http/http-client-auth.interceptor.ts

```diff
@@ -27,12 +27,15 @@ export class HttpClientAuthInterceptor implements HttpInterceptor {
 
   intercept(request: HttpRequest<any>, next: HttpHandler): Observable<HttpEvent<any>> {
     let headers = request.headers.set('Authorization', `Bearer ${this.chefSession.id_token}`);
-    if (this.projects) {
+    const filtered = request.params.get('unfiltered') !== 'true';
+    const params = request.params.delete('unfiltered');
```

Comment:
❓ Is one endpoint used for both filtered and unfiltered requests? If so, fine; if not, we could also have the endpoint ignore an incoming list of projects, I suppose.

Reply:
The intention is that this is a general purpose mechanism; not designed for any specific endpoint.

Reply:
Yeah nevermind. I don't know what I've been thinking -- this is the client req interceptor, after all. 😄